vendor/assets/javascripts/brainstem/brainstem-model.coffee

```diff
@@ -45,7 +45,7 @@ class window.Brainstem.Model extends Backbone.Model
   @parse: (modelObject) ->
     for k,v of modelObject
       # Date.parse will parse ISO 8601 in ECMAScript 5, but we include a shim for now
-      if /\d{4}-\d{2}-\d{2}T\d{2}\:\d{2}\:\d{2}[-+]\d{2}:\d{2}/.test(v)
+      if /^\d{4}-\d{2}-\d{2}T\d{2}\:\d{2}\:\d{2}[-+]\d{2}:\d{2}$/.test(v)
```

Comment:
`^` matches to beginning of the line and `$` matches to end of line character. So line boundary?